diff -up gimagereader-3.4.3/gtk/src/Config.cc.2~ gimagereader-3.4.3/gtk/src/Config.cc
--- gimagereader-3.4.3/gtk/src/Config.cc.2~	2025-08-04 23:30:50.000000000 +0200
+++ gimagereader-3.4.3/gtk/src/Config.cc	2026-03-28 01:46:49.754842532 +0100
@@ -218,7 +218,7 @@ std::string Config::spellingLocation(Loc
 		if (Gio::File::create_for_path(Glib::build_filename(dataDir, "myspell"))->query_exists()) {
 			return Glib::build_filename(dataDir, "myspell");
 		} else {
-			return Glib::build_filename(dataDir, "hunspell");
+			return Glib::build_filename(dataDir, "dict", "ooo");
 		}
 #else
 		return Glib::build_filename(dataDir, "myspell", "dicts");
diff -up gimagereader-3.4.3/gtk/src/MainWindow.cc.2~ gimagereader-3.4.3/gtk/src/MainWindow.cc
--- gimagereader-3.4.3/gtk/src/MainWindow.cc.2~	2025-08-04 23:30:50.000000000 +0200
+++ gimagereader-3.4.3/gtk/src/MainWindow.cc	2026-03-28 01:47:35.799383439 +0100
@@ -539,6 +539,7 @@ void MainWindow::dictionaryAutoinstall(G
 	std::uint32_t xid = gdk_x11_window_get_xid(getWindow()->get_window()->gobj());
 	std::vector<Glib::ustring> files;
 	for (const Glib::ustring& langCulture : m_config->searchLangCultures(code)) {
+		files.push_back("/usr/share/dict/ooo/" + langCulture + ".dic");
 		files.push_back("/usr/share/myspell/" + langCulture + ".dic");
 		files.push_back("/usr/share/hunspell/" + langCulture + ".dic");
 	}
diff -up gimagereader-3.4.3/qt/src/Config.cc.2~ gimagereader-3.4.3/qt/src/Config.cc
--- gimagereader-3.4.3/qt/src/Config.cc.2~	2025-08-04 23:30:50.000000000 +0200
+++ gimagereader-3.4.3/qt/src/Config.cc	2026-03-28 01:48:15.905516013 +0100
@@ -210,7 +210,7 @@ QString Config::spellingLocation(Locatio
 		if (QDir(dataDir.absoluteFilePath("myspell")).exists()) {
 			return dataDir.absoluteFilePath("myspell");
 		} else {
-			return dataDir.absoluteFilePath("hunspell");
+			return dataDir.absoluteFilePath("dict/ooo");
 		}
 #else
 		return dataDir.absoluteFilePath("myspell/dicts");
diff -up gimagereader-3.4.3/qt/src/MainWindow.cc.2~ gimagereader-3.4.3/qt/src/MainWindow.cc
--- gimagereader-3.4.3/qt/src/MainWindow.cc.2~	2025-08-04 23:30:50.000000000 +0200
+++ gimagereader-3.4.3/qt/src/MainWindow.cc	2026-03-28 01:48:00.050464793 +0100
@@ -552,6 +552,7 @@ void MainWindow::dictionaryAutoinstall()
 #ifdef Q_OS_LINUX
 		QStringList files;
 		for (const QString& langCulture : m_config->searchLangCultures(code)) {
+			files.append("/usr/share/dict/ooo/" + langCulture + ".dic");
 			files.append("/usr/share/myspell/" + langCulture + ".dic");
 			files.append("/usr/share/hunspell/" + langCulture + ".dic");
 		}
